### DISCUSSION TITLE
Game crashes when loading into result screen with additional slots

### DISCUSSION THREAD

**Discussion by mRama3415** [Raytwo/ARCropolis#472] [Issues]
Whenever I play a game with an added slot on any character, the game crashes every time when loading the result screen. The css and game loads perfectly fine and the game plays fine, but right as the result screen loads, it crashes only on added slots. 

The odd thing is that if I play with an added slot which was added on the 3.6.0 release of arc, the game and result screen run fine then and on current arc. But any new slots I've added since ~3.8.0 cause the issue with loading into the result screen screen. 

Not sure if anyone else is encountering this issue, but if you have found a fix, please let me know as I'd love to know if I'm dong something wrong, or if its an arc related issue.

**Comment by ElaineLMN** [comment]
I have a similar issue that began happening around the same time, #451. I believe it's an ARC issue, but I haven't been able to get anyone to investigate.
